types/d3-delaunay/index.d.ts

```diff
@@ -143,11 +141,18 @@ export class Delaunay<P> {
      */
     renderPoints(context: Delaunay.MoveContext & Delaunay.ArcContext, radius?: number): void;
 
+    /**
+     * Updates the triangulation after the points have been modified in-place.
+     */
+    update(): Delaunay<P>;
```

Comment:
The return types of both the Delaunay.update and Voronoi.update functions aren't abundantly clear in the documentation, but in the source you can see that they each return themselves.

https://github.com/d3/d3-delaunay/blob/64451c23983fce9f63af3a4c36d80a0bec67a234/src/delaunay.js#L47